modules/kubernetes-addons/aws-for-fluentbit/variables.tf

```diff
@@ -27,6 +27,24 @@ variable "eks_cluster_id" {
   description = "EKS cluster Id"
 }
 
+variable "cw_log_group_name" {
+  type        = string
+  description = "FluentBit CloudWatch Log group name"
+  default     = null
```

Comment:
default

Reply:
yes, default is null. see above response